content/en/docs/quickstart.md

```diff
@@ -43,86 +43,90 @@ notation list --plain-http $IMAGE
+Use `notation key list` to confirm the signing key. Key name with a `*` prefix is the default key.
 
 ```console
-notation sign --plain-http $IMAGE
+notation key list
 ```
 
-Use `notation list` to show the signatures for your container image.
+Use `notation certificate list` to confirm whether the certificate is stored in the trust store.
 
 ```console
-notation list --plain-http $IMAGE
+notation cert list
 ```
 
-Confirm there is one listed, for example:
+## Sign the container image
 
-```output
-$ notation list --plain-http $IMAGE
-sha256:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef
-```
+Use `notation sign` to sign the container image with COSE signature format.
 
-## Verify the signature of an image
+```console
+notation sign --plain-http --signature-format cose $IMAGE
+```
 
-Use `notification verify` to verify any signatures on your container image.
+Use `notation list` to show the signatures associated with the container image.
 
 ```console
-notation verify --plain-http $IMAGE
+notation list --plain-http $IMAGE
 ```
 
-Confirm your container image fails validation. For example:
+Confirm there is one signature, for example:
 
 ```output
-$ notation verify --plain-http $IMAGE
-Error: trust certificate not specified
-2022/09/30 15:24:14 trust certificate not specified
+$ notation list --plain-http $IMAGE
+localhost:5000/net-monitor@sha256:073b75987e95b89f187a89809f08a32033972bb63cda279db8a9ca16b7ff555a
+└── application/vnd.cncf.notary.v2.signature
+    └── sha256:ba3a68a28648ba18c51a479145fca60d96b43dc96c6ab22f412c89ac56a9038b
 ```
 
-This verification fails because you have not trusted the public key for your container image.
+## Create a trust policy
 
-## Add a trusted public key
+In order to verify the container image, you need to configure the trust policy to specify trusted identities which sign the artifacts, and level of signature verification to use. The trust policy is a JSON document with the name `trustpolicy.json` under `$HOME/.config/notation/`.
 
-Add the public from the certificate you generated earlier as a trusted certificate.
+Once the trust policy file is executed as below, it creates one trust policy named `wabbit-networks-images`. This policy only applies to the repository `$REPO`. The level of signatureVerification is `strict`, which enforces a full validation. Any validation failure will fail the whole signature verification process. The trust store used for this policy is named `wabbit-networks.io` of type `ca`, which is created in previous step. Use `notation cert show` to get trust identity info from the subject field of the self-signed certificate.
 
 ```console
-notation cert add --name "wabbit-networks.io" <EXAMPLE_PATH>/notation/localkeys/wabbit-networks.io.crt
+cat <<EOF > $HOME/.config/notation/trustpolicy.json
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-images",
+            "registryScopes": [ "$REPO" ],
```

Comment:
This should be explicit. Is it the registry? Is it the repo minus an image? Does someone have to specify this for every image they want to validate?

Reply:
For what it's worth, with the current version I had to include the image specifically:

```
cat <<EOF > $HOME/.config/notation/trustpolicy.json
{
    "version": "1.0",
    "trustPolicies": [
        {
            "name": "upstream",
            "registryScopes": [ "upstream.azurecr.io/oss/fluxcd/flux" ],
            "signatureVerification": {
                "level" : "strict" 
            },
            "trustStores": [ "ca:trust" ],
            "trustedIdentities": [
                "*"
            ]
        }
    ]
}
EOF
```

If I omitted "flux" I ended up getting errors:

```
ERROR: artifact "upstream.azurecr.io/oss/fluxcd/flux@sha256:0f9ad5df470963f959bb3e0b7abe60e9b4011a636e566a06e883ed7b3812dbc4" has no applicable trust policy

Signature verification failed for all the 0 signatures associated with digest: sha256:0f9ad5df470963f959bb3e0b7abe60e9b4011a636e566a06e883ed7b3812dbc4

Error: artifact "upstream.azurecr.io/oss/fluxcd/flux@sha256:0f9ad5df470963f959bb3e0b7abe60e9b4011a636e566a06e883ed7b3812dbc4" has no applicable trust policy
```

You should allow wild cards here or something. From an end user perspective if I have to build a trust policy that includes every image I want to validate, that becomes an issue when I build any additional images and is a cumbersome experience.

Reply:
`upstream.azurecr.io/*` should work and `["$REPOSITORY/*"]` should be in the example

Reply:
I’ll give that a try!

Reply:
According to the [spec](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#registry-scopes-constraints), the repository URI MUST NOT contain the asterisk character *. You can just use `["*"]` as a global scope for a quick start.

Reply:
Thanks for the additional clarification. So in summary:

Option 1: Trust everything 
Option 2: enumerate every image you will verify?

Reply:
> Thanks for the additional clarification. So in summary:
> 
> Option 1: Trust everything Option 2: enumerate every image you will verify?

@jeremyrickard , you can specify a repo, then all the artifacts under the repo will be verified against the policy. For example, you have images under repo `localhost:5000/net-monitor`
- localhost:5000/net-monitor:v1
- localhost:5000/net-monitor:v2
- localhost:5000/net-monitor:v3

you just need to specify `localhost:5000/net-monitor` for registryScopes parameter.

Reply:
Check out here for more scenarios: https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-policy-schema

Reply:
@yizha1 - yes this is far too much work!  We need to add a backlog item to support all images coming from a registry.